src/vsr/replica.zig

```diff
@@ -3344,6 +3346,36 @@ pub fn ReplicaType(
             }
         }
 
+        // The actual "execution" was handled by the primary when the request was prepared.
+        // Primary makes use of local information to decide whether reconfiguration should be accepted.
+        // Here, we just copy over the result.
+        fn commit_reconfiguration(
+            self: *Self,
+            prepare: *const Message,
+            output_buffer: *align(16) [constants.message_body_size_max]u8,
+        ) usize {
+            assert(self.commit_stage == .setup_client_replies);
+            assert(self.commit_prepare.? == prepare);
+            assert(prepare.header.command == .prepare);
+            assert(prepare.header.operation == .reconfigure);
+            assert(prepare.header.size == @sizeOf(vsr.Header) + @sizeOf(vsr.ReconfigurationRequest));
+            assert(prepare.header.op == self.commit_min + 1);
+            assert(prepare.header.op <= self.op);
+
+            const reconfiguration_request = std.mem.bytesAsValue(
+                vsr.ReconfigurationRequest,
+                prepare.body()[0..@sizeOf(vsr.ReconfigurationRequest)],
+            );
```

Comment:
```suggestion
            );
            assert(reconfiguration_request.result != .reserved);
```

Reply:
Did this assert get missed? I see it in `primary_prepare_reconfiguration` but I think it applies to `commit_reconfiguration` too.

Reply:
indeed, added!